.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+
+# This workflow will build a package using Gradle and then publish it to Sonatype OSSRH
+
+name: Gradle Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Build with Gradle
+        run: gradle build
```

Comment:
@puug does this need to `./gradew` instead? It doesn't look like `setup-java@v2` installs gradle.

Reply:
updated the commands